tests/roles/backend_services/tasks/main.yaml

```diff
@@ -1,3 +1,46 @@
+- name: check if the source SC exists and can be used for the local-storage SC
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get sc {{ storage_class_name }} -o yaml
+  register: local_sc_result
+  failed_when:
+    - local_sc_result.rc != 0
+    - ('Error from server (NotFound)' not in local_sc_result.stdout)
+  changed_when: local_sc_result.rc != 0
+
+- name: create a local-storage SC from the given source SC
+  when: not local_sc_result.changed
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc apply -f - <<EOF
+    {{ local_sc_result.stdout | from_yaml | combine({"reclaimPolicy": storage_reclaim_policy, "resourceVersion": omit, "uuid": omit, "creationTimestamp": omit, "metadata": {"annotations": omit}}) }}
+    EOF
+
+- name: create a local-storage SC from crc-csi-hostpath-provisioner SC
+  when: local_sc_result.changed
```

Comment:
add  # noqa: no-handler

```
when: local_sc_result.changed # noqa: no-handler
```

Reply:
We should fix these linter issues though